src/components/ImageGrid/ImageGrid.tsx

```diff
@@ -66,15 +67,28 @@ export const ImageGrid = ({ openDrawer }: ImageGridProps) => {
     return categories[index];
   };
 
+  const scaleFactor = useSelector(tileSizeSelector);
+
+  const getSize = (scaleFactor: number) => {
+    const width = (230 * scaleFactor).toString() + "px";
+    const height = (185 * scaleFactor).toString() + "px";
+    return { width: width, height: height };
+  };
+
   return (
     <React.Fragment>
       <main className={clsx(classes.main, { [classes.mainShift]: openDrawer })}>
         <Container className={classes.container} maxWidth="md">
-          <GridList className={classes.gridList} cols={4}>
+          <GridList
+            className={classes.gridList}
+            cols={Math.floor(4 / scaleFactor)}
```

Comment:
Maybe the slider should change the `cols` value and set the `width` of `GridListTile` to `100%`.

Reply:
The tiles had weird behaviour when I was testing - height and width are handled separately, so changing cols would scale the width but not the height. I figured it'd be better to just work on width and height together then adjust columns to fit sensibly. We do need to correct some of the excess padding surrounding the grid widget though.